src/features/navigation/SearchStackNavigator/SearchStackNavigator.tsx

```diff
@@ -14,7 +21,52 @@ export const SearchStackNavigator = ({
     <SearchStack.Navigator
       initialRouteName={initialRouteName}
       screenOptions={SEARCH_STACK_NAVIGATOR_SCREEN_OPTIONS}>
-      {SearchScreens}
+      {routes.map(({ name, component }) => (
+        <SearchStack.Screen name={name} key={name} component={component} />
+      ))}
     </SearchStack.Navigator>
   )
 }
+
+const routes: SearchStackRoute[] = [
+  {
+    name: 'SearchLanding',
```

Comment:
on a un enum pour les screens de la recherche

est-ce qu'il ne serait pas mieux de l'utiliser ?

```suggestion
    name: SearchView.Landing,
```